roles/confluent.variables/defaults/main.yml

```diff
@@ -494,10 +516,10 @@ kafka_connect_monitoring_interceptors_enabled: "{{ monitoring_interceptors_enabl
 
 #### KSQLDB Variables ####
 
-### Only use to customize Linux User ksqlDB Service runs with. User must exist on host.
+### Only use to customize Linux User ksqlDB Service runs with. Default user is cp-ksql
```

Comment:
Modify this variable to customize the Linux User that the ksqlDB Service runs with. Default user is cp-ksql.